go/sql/parser_test.go

```diff
@@ -52,23 +51,23 @@ func TestParseAlterStatementWithAutoIncrement(t *testing.T) {
 	for _, statement := range statements {
 		parser := NewAlterTableParser()
 		err := parser.ParseAlterStatement(statement)
-		test.S(t).ExpectNil(err)
-		test.S(t).ExpectEquals(parser.alterStatementOptions, statement)
-		test.S(t).ExpectTrue(parser.IsAutoIncrementDefined())
+		require.NoError(t, err)
+		require.Equal(t, statement, parser.alterStatementOptions)
+		require.True(t, parser.IsAutoIncrementDefined())
 	}
 }
 
-func TestParseAlterStatementTrivialRenames(t *testing.T) {
+func TestParseAlterStatementrivialRenames(t *testing.T) {
 	statement := "add column t int, change ts ts timestamp, CHANGE f `f` float, engine=innodb"
 	parser := NewAlterTableParser()
 	err := parser.ParseAlterStatement(statement)
-	test.S(t).ExpectNil(err)
-	test.S(t).ExpectEquals(parser.alterStatementOptions, statement)
-	test.S(t).ExpectFalse(parser.HasNonTrivialRenames())
-	test.S(t).ExpectFalse(parser.IsAutoIncrementDefined())
-	test.S(t).ExpectEquals(len(parser.columnRenameMap), 2)
-	test.S(t).ExpectEquals(parser.columnRenameMap["ts"], "ts")
-	test.S(t).ExpectEquals(parser.columnRenameMap["f"], "f")
+	require.NoError(t, err)
+	require.Equal(t, statement, parser.alterStatementOptions)
+	require.False(t, parser.HasNonTrivialRenames())
+	require.False(t, parser.IsAutoIncrementDefined())
+	require.Equal(t, 2, len(parser.columnRenameMap))
```

Comment:
**nit**: there also is `require.Len()`

Reply:
Nice! Will apply that change to all the cases I can find.

Reply:
Done!